presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClientModule.java

```diff
@@ -97,8 +96,7 @@ public static CassandraSession createCassandraSession(
         requireNonNull(config, "config is null");
         requireNonNull(extraColumnMetadataCodec, "extraColumnMetadataCodec is null");
 
-        Cluster.Builder clusterBuilder = Cluster.builder()
-                .withProtocolVersion(ProtocolVersion.V3);
+        Cluster.Builder clusterBuilder = Cluster.builder();
```

Comment:
Our requirement is to check for Cassandra version and based on that allow IN predicate push-down. But I could see protocol version is not the good solution to check this. Even for Cassandra version > 2.2.X, we can configure protocol version 3 during initialization then IN predicate will not work. 

Do you have any alternate suggestion to handle this situation? How about getting the Cassandra version with the below query?
`select release_version from system.local where key = 'local';`